### PR TITLE
PMM-6567 pg_stat_monitor

### PR DESCRIPTION
**Pull request by JiriCtvrtka**
[PMM-6567](https://jira.percona.com/browse/PMM-6567)

- [x] Build: https://github.com/Percona-Lab/pmm-submodules/pull/1104

- [x] Links to other linked pull requests (optional).
pmm-api-tests: https://github.com/Percona-Lab/pmm-api-tests/pull/124
pmm: https://github.com/percona/pmm/pull/584
pmm-submodules: https://github.com/Percona-Lab/pmm-submodules/pull/1104
pmm-agent: https://github.com/percona/pmm-agent/pull/186

---
- [x] Tests passed.
- [x] Feature build pass.
- [x] (Re)requested review.
- [ ] Fix conflicts with target branch.
- [ ] Update API dependency.
- [ ] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

### CODE REVIEW

**Review by BupycHuk — CHANGES_REQUESTED** [review]
Everything is looks ok except two things.
1. You forgot about disable_query_examples
2. Inventory API was out of scope of this task, but let's keep it :)

File: services/converters.go [hunk 1]
```diff
@@ -324,6 +324,19 @@ func ToAPIAgent(q *reform.Querier, agent *models.Agent) (inventorypb.Agent, erro
 			TlsSkipVerify: agent.TLSSkipVerify,
 		}, nil
 
+	case models.QANPostgreSQLPgStatMonitorAgentType:
+		return &inventorypb.QANPostgreSQLPgStatMonitorAgent{
```
Comment (BupycHuk) [inline services/converters.go:328]: `disable_query_examples` missed

**Review by BupycHuk — CHANGES_REQUESTED** [review]

File: services/converters.go (hunk 1 above)
Comment (JiriCtvrtka) [reply services/converters.go:328]: Right, thanks.

**Comment by AlekSi** [comment]
There is a conflict now

**Comment by JiriCtvrtka** [comment]
FB: https://github.com/Percona-Lab/pmm-submodules/pull/1104

PRs:
PMM - https://github.com/percona/pmm/pull/584
PMM managed - https://github.com/percona/pmm-managed/pull/457
PMM API tests - https://github.com/Percona-Lab/pmm-api-tests/pull/124
PMM agent - https://github.com/percona/pmm-agent/pull/186

**Review by BupycHuk — APPROVED** [review]

File: Gopkg.toml [hunk 2]
```diff
@@ -16,7 +16,7 @@ required = [
 
 [[constraint]]
   name = "github.com/percona/pmm"
-  branch = "PMM-2.0"
+  branch = "PMM-6567-pg-monitor"
```
Comment (askomorokhov) [inline Gopkg.toml:19]: Please change the branch back before marge.

**Review by askomorokhov — APPROVED** [review]
Looks good to me.

**Comment by AlekSi** [comment]
@JiriCtvrtka please resolve merge conflicts

**Pull request merged**